lrlex/src/lib/parser.rs

```diff
@@ -365,15 +366,15 @@ where
             let j = self.parse_ws(i)?;
```

Comment:
So the main case is where this is hitting A string like "   \n" it includes the newline in the return value `j`.
Perhaps this should also consider changing to use the `parse_spaces` that I added for the other issue to avoid matching that newline.